src/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -63,7 +62,7 @@ describe("logSingleWord", () => {
 		const position: Position = { fromIndex: 0, toIndex: 4 };
 		const issues = logSingleWord("Hello", position);
 		expect(issues).toEqual({
-			code: IssueCode.VocabularyIssue,
+			code: InvalidWord,
```

Comment:
No, that won't work that way. `InvalidWord` as you imported it is a `{ message: string, severity: string }` and not a `string` as should be expected!

Reply:
@Korosensei42